R/utils.R

```diff
@@ -68,6 +69,7 @@ h_record_all_output <- function(expr, remove = list()) {
 #'
 #' @export
 free_cores <- function() {
+  .Deprecated("parallelly::availableCores(omit = 1)")
```

Comment:
suggest to follow https://lifecycle.r-lib.org/articles/manage.html so `lifecyle::deprecate_warn()` here instead

Reply:
ok, I think I got it in and adjusted the test

Reply:
btw, I am prospectively setting the depreciation version to 0.1.6 since we are in 0.1.5.9xxx now, correct?

Reply:
perfect